### PR TITLE
[stable/traefik] Enable HTTP authentication for the dashboard

### PR DESCRIPTION
**Pull request by tolbrino**
Also allow configuration of the dashboard's service address and whether the Ingress should be generated.

### CODE REVIEW

**Comment by tolbrino** [comment]
I signed it!

**Comment by lachie83** [comment]
cc @krancour

File: stable/traefik/README.md [hunk 1]
```diff
@@ -87,6 +87,9 @@ The following tables lists the configurable parameters of the Traefik chart and
 | `acme.persistence.size`         | Minimum size of the volume requested                                 | `1Gi`                                     |
 | `dashboard.enabled`             | Whether to enable the Traefik dashboard                              | `false`                                   |
 | `dashboard.domain`              | Domain for the Traefik dashboard                                     | `traefik.example.com`                     |
+| `dashboard.address`             | Address for the Traefik dashboard service                            | `:8080`                                   |
+| `dashboard.public`              | Whether the Traefik dashboard should be exposed via an Ingress       | `true`                                    |
+| `dashboard.http_auth`           | List of user/password combinations to be used via HTTP Auth          | none                                      |
```
Comment (unguiculus) [inline stable/traefik/README.md]: Can we stick with camel case to be consistent with the other properties?
-> `dashboard.httpAuth`

Comment (unguiculus) [inline stable/traefik/README.md]: Can we stick with camel case to be consistent with the other properties?
-> `dashboard.httpAuth`

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/traefik/README.md (hunk 1 above)
Comment (tolbrino) [reply stable/traefik/README.md]: Definitely, will fix it.

**Comment by krancour** [comment]
This is a duplicate of #414. This may have been submitted first, but #414 has been more thoroughly reviewed, requested changes already applied, etc.

**Comment by tolbrino** [comment]
@krancour Agreed, I could strip the http auth part from this PR and leave that to #414. Then only the `address` and `public` parameters would be handled here.

**Comment by krancour** [comment]
@tolbrino ya... let's take the basic auth stuff out of this PR and consider what remains on its own merit. tbh, I haven't looked to closely at those others changes yet.

**Comment by lachie83** [comment]
@tolbrino @krancour What would you like me to do with this?

**Comment by tolbrino** [comment]
@lachie83 I forgot to adapt the PR so far. Will do so in the coming days, then you can review it again.

**Comment by lachie83** [comment]
@tolbrino can you please provide a status update?

**Comment by seanknox** [comment]
Closing due to inactvitiy. @tolbrino feel free to re-open.

**Pull request closed without merging**